src/docfx/config/ops/OpsAccessor.cs

```diff
@@ -234,111 +161,94 @@ private async Task<string> FetchValidationRules(string routePath, string reposit
             }
         }
 
-        private async Task<string> FetchTaxonomies(DocsEnvironment? environment = null)
+        private async Task<string> Fetch(string url, Dictionary<string, string>? headers = null, string? value404 = null)
```

Comment:
I refactored this file a little bit to ensure all GET APIs ended up into this `Fetch` method.